mrmustard/utils/wigner.py

```diff
@@ -27,7 +28,7 @@ def wigner_discretized(rho, qvec, pvec, hbar=settings.HBAR):
 
     Args:
         rho (complex array): the density matrix of the state in Fock representation
-        xvec (array): array of discretized :math:`x` quadrature values
+        qvec (array): array of discretized :math:`q` quadrature values
```

Comment:
Are we going to express all xp related to qp? (It works for me. Just want to know if I meet it in the future, I will change them)

Reply:
I changed the name of the argument here because the function expects q (i.e. x/sqrt(2hbar)). There is no strict convention: some functions need q some x.